src/components/App/ExtensibilityRoutes.js

```diff
@@ -97,7 +140,7 @@ export const createExtensibilityRoutes = (cr, language) => {
         exact
         element={
           <Suspense fallback={<Spinner />}>
-            <ColumnWrapper resourceType={urlPath} />
+            <ColumnWrapper resourceType={urlPath} cr={cr} />
```

Comment:
`cr` is not so clear to understand what this prop is passing, please rename this